### PR TITLE
Modified logger to store notification message in one more place

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat @nthomas-redhat please review

**Comment by GowthamShanmugam** [comment]
verfied

File: tendrl/commons/logger.py [hunk 1]
```diff
@@ -96,6 +99,22 @@ def push_event(self):
...
+            priority=self.message.priority,
+            publisher=self.message.publisher,
+            node_id=self.message.node_id,
+            payload=self.message.payload,
+            cluster_id=self.message.cluster_id,
+            job_id=self.message.job_id,
+            flow_id=self.message.flow_id,
+            parent_id=self.message.parent_id,
+            caller=self.message.caller
+        ).save()
```
Comment (r0h4n) [inline tendrl/commons/logger.py]: no ttl for save?

File: tendrl/commons/logger.py [hunk 2]
```diff
@@ -23,6 +23,9 @@ def __init__(self, message):
             self._logger(self.push_operation())
         else:
             self.push_event()
+            if self.message.priority == "notice":
+                # push notification
+                self.push_notification()
```
Comment (r0h4n) [inline tendrl/commons/logger.py]: Please return here, no need to further duplicate the notification in /messages/events

**Comment by GowthamShanmugam** [comment]
verfied

**Pull request merged**